libcontainerd/container_windows.go

```diff
@@ -20,6 +21,8 @@ type container struct {
 	// but can be called from the RestartManager context which does not
 	// otherwise have access to the Spec
 	ociSpec Spec
+
+	stopped sync.WaitGroup
```

Comment:
Are you sure about that? There are an undefined number of callers we have to wait for - one for the first process started through docker run, and one for each possible exec. A waitgroup makes this a lot clearer IMO...